Sources/PreventSleep.swift

```diff
@@ -58,7 +58,7 @@ public class PreventSleep {
     
     // Allow the computer to go to sleep
     // Returns whether the power assertion was successful or not.
-    @objc public func allowSleep() -> Bool {
+    @objc @discardableResult open func allowSleep() -> Bool {
```

Comment:
Added @discardableResult so we aren't forced to use the result, otherwise the compiler was  throw warnings at line 32.